spec/models/like_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
## [OPTIONAL] 
- [ ] Kindly correct the modal name it, should be `Like` and not `User`

Reply:
Ok thaks!